src/components/DatePicker/DatePickerProps.ts

```diff
@@ -0,0 +1,7 @@
+type SizeType = 'small' | 'medium' | 'large' | undefined;
```

Comment:
```suggestion
type SizeType = 'small' | 'medium' | 'large';
```

ini gaperlu undefined lagi kalo udh tanda tanya di interfacenya